src/validation.cpp

```diff
@@ -5107,6 +5106,7 @@ void ChainstateManager::Unload()
 
     m_failed_blocks.clear();
     m_blockman.Unload();
+    pindexBestHeader = nullptr;
```

Comment:
In commit "Clear pindexBestHeader in ChainstateManager::Unload()" (b0b7d756ba6f943f3292734f4fa35a16b8cbabde)

Note: This seems safe because because the `pindexBestHeader = nullptr` call is moving from `UnloadBlockIndex` function to the `ChainstateManager::Unload()` method, which is called by `UnloadBlockIndex` and no other callers.